src/components/molecules/AdminInput/AdminInput.tsx

```diff
@@ -29,8 +31,17 @@ export const AdminInput: React.FC<AdminInputProps> = ({
     [label, name]
   );
 
-  return (
-    <p className="AdminInput">
+  return hidden ? (
+    <input
+      {...inputProps}
+      className="AdminInput AdminInput--hidden AdminInput__input"
+      name={name}
+      ref={register}
+      id={id}
```

Comment:
```suggestion
      ref={register}
```
You can delete these as they are spread on the top

Reply:
They aren't.

```tsx
  name,
  label,
  subtext,
  register,
  errors,
  hidden: isHidden,
  ...inputProps
```